docs: add guide for importing components from JLCPCB

khopade-works opened the pull request:
# Description
This PR adds documentation for importing components from JLCPCB using RunFrame in TSCircuit. The guide provides users with a comprehensive walkthrough of the import process and related features.

## Changes
- Added new documentation file: `docs/guides/importing-modules-and-chips/importing-from-jlcpcb.md`
- Includes detailed instructions for using RunFrame to import JLCPCB components
- Covers key features like direct integration, smart search, and real-time updates
- Provides best practices and troubleshooting guidance

## Related Issues
Closes #40
/claim #40

Anshgrover23 reviewed (changes requested):
@seveibar i think the issue is about to add different tscircuit repos.
so yours pov??

Ayushjhawar8 commented on tutorials/contributing.mdx:
````diff
@@ -0,0 +1,197 @@
+3. **Development Process**
+   ```bash
+   # Fork and clone the repository
+   git clone https://github.com/<your-username>/<repository>
````
thanks arnav fixing it asap

Ayushjhawar8 replied:
thanks arnav for review, but its actually for cloning the fork into the local system, so user might need there username 🙂

seveibar reviewed (changes requested):
I don't like the length of it, i feel like a table with an overview of the main repos (linked), with a badge indicating the number of open issues in one column and a description of the repo would be better

maybe a column with tags like “skillset”

never include basic instructions like how to clone etc.

Ayushjhawar8 commented:
https://github.com/user-attachments/assets/b9344caa-f724-4a52-a6fb-cfac735a3d45

aayush1root committed:
docs: add guide for importing components from JLCPCB

Add comprehensive documentation for importing components from JLCPCB using RunFrame, including step-by-step instructions, key features, best practices, and troubleshooting tips.

techmannih reviewed (changes requested):
@Apocalypse96 Thanks for your PR – it looks good!

You don't need to create a new file. The file importing-from-jlcpcb.md already exists. You just need to add a new section to it titled "Runframe Imports".

Make sure to include a video guide in that section, as it's required for the issue.

You can also refer to this page for additional context:
https://docs.tscircuit.com/guides/importing-modules-and-chips/importing-from-jlcpcb

aayush1root committed:
docs: update JLCPCB import guide with Runframe details and improve formatting

Removed the old importing guide and enhanced the existing documentation for importing components from JLCPCB using Runframe. Added step-by-step instructions, a video guide, and improved formatting for clarity.

khopade-works commented:
Hey @techmannih , Done the changes you asked

techmannih commented:
@Apocalypse96 Nice work!

Could you please share the video here?

You can also check the deployed version, video is not working:
https://docs-4vyg8q8f1-tscircuit.vercel.app/guides/importing-modules-and-chips/importing-from-jlcpcb

khopade-works commented:
Hey @techmannih , I am bit confused , Do i have to make a video or you have an existing video , which we have to display it there ??

techmannih commented:
Yeah, you have to create a short video, in that video you need to explain how to export jlcpcb part using runframe and add it here

Cc @seveibar

imrishabh18 reviewed (changes requested):
Where is the video?

Anshgrover23 reviewed (changes requested):
@imrishabh18 i am curious do we have resolved the issue of component files not showing when importing components from jlcpcb then how he is adding docs?

imrishabh18 commented:
@Anshgrover23 I don't know, if it isn't resolved in prod then we won't be getting the video for it. Hence, this won't be merged until then

Anshgrover23 commented:
@imrishabh18 ohh sorry, my fault i missed your pr. yes the issue is resolved. @Apocalypse96 kindly put the required video?

Anshgrover23 commented:
@Apocalypse96 issue resolved you can close your PR now.
